overviewwidget.cpp

```diff
@@ -88,3 +93,20 @@ void OverviewWidget::modelUpdated()
 {
     m_treeView->expandAll();
 }
+
+void OverviewWidget::onItemActivated(const QModelIndex &index)
+{
+    if (!index.isValid())
+        return;
+
+    ModelTreeNode *node = static_cast<ModelTreeNode *>(index.internalPointer());
+
+    // it is assumed, that if item has children, user wants to expand it, even though sometimes it could be jumped to.
```

Comment:
I'd have to check how it behaves - there's a separate icon for expanding, so maybe this if should be removed